modules/exploits/linux/local/pkexec.rb

```diff
@@ -66,6 +66,24 @@ def executable_path
     @executable_path
   end
 
+  def check
+    # version can be nil
+    version = cmd_exec('pkexec --version').split.last
+
+    # version can be a string, so we check it
+    if version.nil? || !Gem::Version.correct?(version)
```

Comment:
I don't see ```Gem::Version.correct?``` being used anywhere in Framework. We already use ```Gem::Version.new``` but without any error checking. I hope this PR changes that trend.